code/game/machinery/slotmachine.dm

```diff
@@ -261,6 +265,8 @@
 				cointype = pick(subtypesof(/obj/item/coin))
 				var/obj/item/coin/C = new cointype(loc)
 				random_step(C, 2, 50)
+				playsound(src, pick(list('sound/machines/coindrop.ogg', 'sound/machines/coindrop2.ogg')), 50, TRUE)
+				sleep(REEL_DEACTIVATE_DELAY)
```

Comment:
this should be a callback timer to a separate proc

Reply:
Like replace these two lines with `addtimer(CALLBACK(src, PROC_REF(coin_sound)))` . And then elsewhere do `/obj/machinery/computer/slot_machine/proc/coin_sound()` with these lines under? If not this you'd have to tell me what exactly